params/config.go

```diff
@@ -724,6 +788,11 @@ func (c *ChainConfig) CheckCompatible(newcfg *ChainConfig, height uint64, isQuor
 	if err != nil {
 		return newCompatError(err.Error(), cBlock, newCfgBlock)
 	}
+	// compare the transitions data between the old and new config
+	err, cBlock, newCfgBlock = isTransitionsConfigCompatible(c, newcfg, bhead)
```

Comment:
so cBlock and newCfgBlock are used for creating a compat error, why not returning this error directly from the isTransiti...Compatible()?

also isTransi...Compatible(), I thought it would return a bool, could it be named checkCompatibilityOfConfigs()?